<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watch and Touch Collaborative Drawing Tests</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="CollaborativeDrawingWidget" name="graphicsView">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear"/>
   <addaction name="actionRedPen"/>
   <addaction name="actionBlackPen"/>
   <addaction name="actionBluePen"/>
   <addaction name="actionEraser"/>
   <addaction name="actionFreehand"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionStraightLine"/>
   <addaction name="actionPenWidthInc"/>
   <addaction name="actionPenWidthDec"/>
   <addaction name="actionConnect"/>
   <addaction name="actionLeave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionRedPen">
   <property name="text">
    <string>Red</string>
   </property>
   <property name="toolTip">
    <string>Red</string>
   </property>
  </action>
  <action name="actionBlackPen">
   <property name="text">
    <string>Black</string>
   </property>
   <property name="toolTip">
    <string>Black</string>
   </property>
  </action>
  <action name="actionBluePen">
   <property name="text">
    <string>Blue</string>
   </property>
   <property name="toolTip">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionPenWidthInc">
   <property name="text">
    <string>PenW++</string>
   </property>
  </action>
  <action name="actionPenWidthDec">
   <property name="text">
    <string>PenW--</string>
   </property>
  </action>
  <action name="actionFreehand">
   <property name="text">
    <string>Freehand</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionStraightLine">
   <property name="text">
    <string>Straight Line</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionListen">
   <property name="text">
    <string>Listen</string>
   </property>
   <property name="toolTip">
    <string>Listen for incoming connections</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionLeave">
   <property name="text">
    <string>Leave</string>
   </property>
   <property name="toolTip">
    <string>Leave Session</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CollaborativeDrawingWidget</class>
   <extends>QGraphicsView</extends>
   <header>collaborativedrawingwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
